scripts/release_prep.sh

```diff
@@ -0,0 +1,122 @@
+#!/bin/bash
+
+set -x
+
+# pre-requisites: install github CLI
+# - github documentation: https://github.com/cli/cli#installation
+# - github is remote 'origin'
+# - PRs use test section LAST with heading "## Tests"
+# - ALL build and release PRs start with "build: "
+#
+# To run, from root of project:
+# bash scripts/release_prep.sh
+
+has_local_changes=$(git status --porcelain --untracked-files=no --ignored=no)
+if [[ ${has_local_changes} ]]; then
+  set +x
+  echo ==========
+  echo "ABORT: You have local modifications. Please stash or commit changes and run again."
+  echo ==========
+  exit 1
+fi
+
+git fetch --all --tags
```

Comment:
@timotheeg bump on this tho!

Reply:
Hmm, I reckon we shouldn't reset all local tags (in case you have whatever tags specific to your local env), but all tags from origin should indeed overwrite conflicting local tags, yes! (origin is source of truth!)

It's not blocking though, when @dcshzj run the script earlier, there was a conflict for a tag `v0.69.0` he had locally that was different from origin, but the script just moved on anyway. The local tags are not used by the script, the only time things could go wrong if there's already a tag for the one the script try to create. 

For that case, there's a `--recut` option btw.

So hmmm, it's not the purpose of the script to clean your local tags, but I suppose it shouldn't be hard to add that cleanup as well 🤔

Reply:
done: https://github.com/isomerpages/isomercms-backend/pull/1209/commits/1382944fe43676880fe8a8565597c3be6d7b74e9